persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_21/target/persist_db_scripts.sql

```diff
@@ -0,0 +1,20 @@
+DROP TABLE IF EXISTS AidPackageOrderItem;
+
+DROP TABLE IF EXISTS MedicalNeed;
+CREATE TABLE MedicalNeed (
+	needId INT NOT NULL AUTO_INCREMENT,
+	beneficiaryId INT NOT NULL,
+	period DATETIME NOT NULL,
+	urgency VARCHAR(191) NOT NULL,
+	quantity INT NOT NULL,
+	PRIMARY KEY(needId)
+);
+
+CREATE TABLE AidPackageOrderItem (
+	id INT NOT NULL AUTO_INCREMENT,
+	needId INT UNIQUE,
+	CONSTRAINT FK_AIDPACKAGEORDERITEM_MEDICALNEED_0 FOREIGN KEY(needId) REFERENCES MedicalNeed(needId),
+	quantity INT NOT NULL,
+	totalAmount INT NOT NULL,
+	PRIMARY KEY(id)
+);
```

Comment:
add extra line at the end

Reply:
I added a new line but this file is a Autogenerated one by the compiler plugin